tools/ui-components/src/row/row.tsx

```diff
@@ -0,0 +1,10 @@
+import React from 'react';
+
+import { RowProps } from './types';
+
+export const Row = ({ className, children }: RowProps) => {
+  if (!className) {
+    className = '';
+  }
+  return <div className={`mx-[-15px] ${className}`}>{children}</div>;
```

Comment:
```suggestion
  if (!className) {
    className = '';
  }
  return <div className={`mx-[-15px] ${className ?? ''}`}>{children}</div>;
```
Nitpick :man_shrugging:

Reply:
there should be a linter for these 🤔

Reply:
no linter rule please, if statement are easier to read than the other one

Reply:
In isolation, arguably, but even then `classname = classname ?? ''` is clearer. However, the broader point is that it's unhelpful to force readers to backtrack when reading code. If I have to keep a mental stack, that's effort. If it's unnecessary, then we should fix it :man_shrugging:

Reply:
true when it's unnecessary, but linter role force us to deal with the necessary the wrong way.

Reply:
Oh, yeah, I agree. It would be an impressive linter that could make this kind of subjective call!